src/backend/web/templates/admin/main_landing.html

```diff
@@ -0,0 +1,39 @@
+{% extends "admin/base.html" %}
+
+{% block title %}TBA - Main Landing Page{% endblock %}
+
+{% block content %}
+
+<h1>TBA Main Landing Page</h1>
+
+<h2>Page Type</h2>
+<p>Choose which landing page to use</p>
+<form id="landing_form" method="post">
+<select name="landing_type">
+  <option value=""></option>
+{% for enum, name in landing_types.items() %}
+  <option value="{{ enum.value }}" {% if current_landing == enum %}selected{% endif %}>{{ name }}</option>
+{% endfor %}
+</select>
+
+<h2>Page Properties</h2>
+<table class="table table-slanding_typetriped">
+<tr><th>Key</th><th>Value</th></tr>
+{% for key, value in current_config.items() %}
+  {% if key != 'current_landing' %}
+  <tr>
+    <td>{{ key }}</td>
+    {% if value is boolean %}
+    <td><input type="checkbox" name="{{ key }}"{% if value %} checked{% endif %}/></td>
+    {% else %}
+    <td><input name="{{ key }}" value="{{ value }}" placeholder="Value" /></td>
+    {% endif %}
+  </tr>
+  {% endif %}
+{% endfor %}
+</table>
+
```

Comment:
I removed the "add key" situation since it's a bit of an antipattern now that our sitevars are typed in code. The template and code are still chill with adding these on the fly, but it's not like we add key/values for the landing page and don't ship a landing page code change. So this functionality didn't seem necessary to me.